plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -501,6 +499,114 @@ void GSRendererDX11::EmulateChannelShuffle(GSTexture** rt, const GSTextureCache:
 	}
 }
 
+void GSRendererDX11::EmulateBlending()
+{
+	// Partial port of OGL SW blending. Currently only works for accumulation blend or when
+	// m_sw_blending >= ACC_BLEND_HIGH_D3D11 since it has not been fully tested. There is no check
+	// currently for primitive overlap like in OGL but it should give better results for post-processing effects
+	// that are done by non-overlapping sprites.
+	const GIFRegALPHA& ALPHA = m_context->ALPHA;
+	bool sw_blending         = false;
+
+	m_om_bsel.abe = PRIM->ABE || (PRIM->AA1 && m_vt.m_primclass == GS_LINE_CLASS);
+	m_om_bsel.a = ALPHA.A;
+	m_om_bsel.b = ALPHA.B;
+	m_om_bsel.c = ALPHA.C;
+	m_om_bsel.d = ALPHA.D;
+
+	if (m_env.PABE.PABE)
+	{
+		if (m_om_bsel.a == 0 && m_om_bsel.b == 1 && m_om_bsel.c == 0 && m_om_bsel.d == 1)
+		{
+			// this works because with PABE alpha blending is on when alpha >= 0x80, but since the pixel shader
+			// cannot output anything over 0x80 (== 1.0) blending with 0x80 or turning it off gives the same result
+
+			m_om_bsel.abe = 0;
+		}
+		else
+		{
+			//Breath of Fire Dragon Quarter triggers this in battles. Graphics are fine though.
+			//ASSERT(0);
+		}
+	}
+
+	// Accumulation blend
+	uint8 blend_index  = uint8(((ALPHA.A * 3 + ALPHA.B) * 3 + ALPHA.C) * 3 + ALPHA.D);
+	int blend_flag = GSDevice11::m_blendMapD3D11[blend_index].flags;
+
+	bool impossible_or_free_blend =
+		(blend_flag & (BLEND_A_MAX|BLEND_NO_BAR|BLEND_ACCU))
+		|| m_bind_rtsample;
```

Comment:
Maybe we can split m_bind_rtsample in two. m_bind_rtsample_one, and m_bind_rtsample_full.

m_bind_rtsample_one for unsafe fbmask without including it in blending.
m_bind_rtsample_full for sw emulated fbmask will also be checked here.